pkg/kubelet/dockertools/manager.go

```diff
@@ -605,6 +605,7 @@ func (dm *DockerManager) runContainer(
 			MemorySwap: -1,
 			CPUShares:  cpuShares,
 		},
+		ShmSize:     1024 * 1024 * 64,
```

Comment:
I don't think engine-api needs omitempty on ShmSize. At least not for right now. The issue we're hitting is only on Fedora 23's current version of Docker, which doesn't accept a ShmSize of 0. I think a combination of fixing Fedora 23's Docker, updating Kube to have a default value (which can be the same as Docker's default 64k), and probably making it configurable in the pod/container spec would be the best path forward.